tests/BUILD.txt

```diff
@@ -0,0 +1,7 @@
+Building aarch64-w64-mingw32 tests on Linux or WSL
+
+cmake -S . -B build
+cmake --build build
+
+it generates Arm64 Windows executable
+build/aarch64_mingw_tests
```

Comment:
It's quite confusing that the generated `build` folder contains `bin` folder but the actual produced binary is in the `build` folder and `bin` is empty.

Reply:
Done